pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: postgrest
 description: PostgREST client for Dart. This library provides an ORM interface to PostgREST.
-version: 0.0.7
+version: 0.0.8
```

Comment:
Version should be 1.0.0, or at least 0.1.0 as suggested by [Flutter Team](https://dart.dev/null-safety/migration-guide#package-version)

Reply:
I originally did that, but changed it at the request of @phamhieu. I think the reasoning was that this is still a pretty early dev version of a package, and since it hasn't hit `0.1.0` or `1.0.0` then breaking changes are to be expected.

Reply:
There will be incoming breaking changes like [148](https://github.com/supabase/postgrest-js/issues/148) and [161](https://github.com/supabase/postgrest-js/issues/161)